Backport of IHP-GmbH/ihp-sg13cmos5l 224b8dc282d2bddf01eb2588fd5217f9e9c4dd17

--- a/libs.tech/klayout/tech/drc/rule_decks/geometry/3_1_offgrid.drc
+++ b/libs.tech/klayout/tech/drc/rule_decks/geometry/3_1_offgrid.drc
@@ -43,19 +43,14 @@
     OFFGRID_LAYERS = {
         # ACTIV
         "activ_drw"    => activ_drw,
-        "activ_pin"    => activ_pin,
         "activ_mask"   => activ_mask,
         "activ_filler" => activ_filler,
-        "activ_nofill" => activ_nofill,
         "activ_opc"    => activ_opc,
         "activ_iopc"   => activ_iopc,
-        "activ_noqrc"  => activ_noqrc,
 
         # GATPOLY
         "gatpoly_drw"    => gatpoly_drw,
-        "gatpoly_pin"    => gatpoly_pin,
         "gatpoly_filler" => gatpoly_filler,
-        "gatpoly_nofill" => gatpoly_nofill,
         "gatpoly_opc"    => gatpoly_opc,
         "gatpoly_iopc"   => gatpoly_iopc,
         "gatpoly_noqrc"  => gatpoly_noqrc,
@@ -68,10 +63,8 @@
 
         # METAL1
         "metal1_drw"     => metal1_drw,
-        "metal1_pin"     => metal1_pin,
         "metal1_mask"    => metal1_mask,
         "metal1_filler"  => metal1_filler,
-        "metal1_nofill"  => metal1_nofill,
         "metal1_slit"    => metal1_slit,
         "metal1_opc"     => metal1_opc,
         "metal1_noqrc"   => metal1_noqrc,
@@ -81,17 +74,14 @@
 
         # PASSIV
         "passiv_drw"    => passiv_drw,
-        "passiv_pin"    => passiv_pin,
         "passiv_pillar" => passiv_pillar,
         "passiv_sbump"  => passiv_sbump,
         "passiv_pdl"    => passiv_pdl,
 
         # METAL2
         "metal2_drw"     => metal2_drw,
-        "metal2_pin"     => metal2_pin,
         "metal2_mask"    => metal2_mask,
         "metal2_filler"  => metal2_filler,
-        "metal2_nofill"  => metal2_nofill,
         "metal2_slit"    => metal2_slit,
         "metal2_opc"     => metal2_opc,
         "metal2_noqrc"   => metal2_noqrc,
@@ -109,10 +99,8 @@
 
         # METAL3
         "metal3_drw"     => metal3_drw,
-        "metal3_pin"     => metal3_pin,
         "metal3_mask"    => metal3_mask,
         "metal3_filler"  => metal3_filler,
-        "metal3_nofill"  => metal3_nofill,
         "metal3_slit"    => metal3_slit,
         "metal3_opc"     => metal3_opc,
         "metal3_noqrc"   => metal3_noqrc,
@@ -122,9 +110,7 @@
 
         # WELLS
         "nwell_drw"   => nwell_drw,
-        "nwell_pin"   => nwell_pin,
         "pwell_drw"   => pwell_drw,
-        "pwell_pin"   => pwell_pin,
         "pwell_block" => pwell_block,
 
         # SUBSTRATE / VIA3
@@ -133,10 +119,8 @@
 
         # METAL4
         "metal4_drw"     => metal4_drw,
-        "metal4_pin"     => metal4_pin,
         "metal4_mask"    => metal4_mask,
         "metal4_filler"  => metal4_filler,
-        "metal4_nofill"  => metal4_nofill,
         "metal4_slit"    => metal4_slit,
         "metal4_opc"     => metal4_opc,
         "metal4_noqrc"   => metal4_noqrc,
@@ -147,10 +131,8 @@
         # TOPVIA1 / TOPMETAL1
         "topvia1_drw"      => topvia1_drw,
         "topmetal1_drw"    => topmetal1_drw,
-        "topmetal1_pin"    => topmetal1_pin,
         "topmetal1_mask"   => topmetal1_mask,
         "topmetal1_filler" => topmetal1_filler,
-        "topmetal1_nofill" => topmetal1_nofill,
         "topmetal1_slit"   => topmetal1_slit,
         "topmetal1_noqrc"  => topmetal1_noqrc,
         "topmetal1_res"    => topmetal1_res,
@@ -160,7 +142,6 @@
         # OTHER CMOS5L LAYERS
         "thickgateox_drw" => thickgateox_drw,
         "polyres_drw"     => polyres_drw,
-        "polyres_pin"     => polyres_pin,
         "antmetal1_drw"   => antmetal1_drw,
         "thinfilmres_drw" => thinfilmres_drw,
         "lbe_drw"         => lbe_drw,
